src/css/base/utility-classes.scss

```diff
@@ -0,0 +1,1767 @@
+@import '../core.scss';
+
+// base utility classes, primarily for documentation and prototyping
+
+$u-percentages: '5', '10', '20', '25', '30', '33', '40', '50', '60', '66', '70', '75', '80', '90', '100';
+$u-percentage-suffix: 'ct';
+$u-percentages-s: '5%', '10%', '20%', '25%', '30%', '33.3333333333%', '40%', '50%', '60%', '66.6666666666', '70%', '75%', '80%', '90%', '100%';
+$u-pixels: '1px', '2px';
+$u-gridscale: '05', '1', '105', '2', '3', '4', '5', '6', '7', '8', '9', '10', '15', '20', '25', '30', '40', '50';
```

Comment:
I don't think this is true. `w50` should be 50xgrid width. 50% width would be `w50ct`.